### PR TITLE
Fix a few issues reported by Lint

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: app/src/main/java/com/ruesga/rview/SearchActivity.java [hunk 1]
```diff
@@ -657,8 +658,8 @@ private void requestCustomSuggestions(String filter) {
     private CharSequence performFilterHighlight(Suggestion suggestion) {
         Spannable span = Spannable.Factory.getInstance().newSpannable(suggestion.mSuggestionText);
         int pos = 0;
-        final String suggestionNoCase = suggestion.mSuggestionText.toLowerCase();
-        final String filterNoCase = suggestion.mFilter.toLowerCase();
+        final String suggestionNoCase = suggestion.mSuggestionText.toLowerCase(Locale.US);
+        final String filterNoCase = suggestion.mFilter.toLowerCase(Locale.US);
```
Comment (jruesga) [inline app/src/main/java/com/ruesga/rview/SearchActivity.java]: This two strings came from user input, so Locale.US could not be the right locale, specially when user is searching for user accounts. I suggest to use the current locale from Configuration.locale, and concretely use AndroidHelper.getCurrentLocale() to deal with the new multilocale methods.

Comment (sschuberth) [inline app/src/main/java/com/ruesga/rview/SearchActivity.java]: Done.

**Comment by sschuberth** [comment]
> This doesn't fit the lint error warning, and the original code is properly scaped.

Correct that it does not make the Lint warning go away, I've clarified the commit message in that regard. But I believe it's not correct that the original code was correctly escaped. As the [linked StackOverflow answer](https://stackoverflow.com/a/38777594/1127485) shows, only the opening angle bracket should be escaped.

**Comment by jruesga** [comment]
Both sentence are ok. What makes the build fails is the `< ` instead of `&lt;`, because that breaks the xml parsing. The second can be represented in the two ways. Since the first one need to be escaped, I opted by escape the second one too. I prefer to keep it as it is, and disable the lint warning with an inline annotation.

**Comment by sschuberth** [comment]
> I prefer to keep it as it is, and disable the lint warning with an inline annotation.

Done.

**Comment by jruesga** [comment]
Thanks

**Pull request merged**